cmd/build.go

```diff
@@ -15,7 +15,7 @@ import (
 	"github.com/pkg/errors"
 	"github.com/spf13/cobra"
 	"github.com/spf13/pflag"
-	"gopkg.in/yaml.v2"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Just curious about this change: did something require this import be explicitly named?

Reply:
This happened when `goimports` ran on one of my machines :man_shrugging: